programs/lazy-distributor/src/token_metadata.rs

```diff
@@ -44,3 +43,108 @@ impl anchor_lang::Owner for Metadata {
     ID
   }
 }
+
+use anchor_lang::prelude::*;
+
+#[derive(AnchorSerialize, AnchorDeserialize, PartialEq, Eq, Debug, Clone)]
+pub enum TokenProgramVersion {
+  Original,
+  Token2022,
+}
+
+#[derive(AnchorSerialize, AnchorDeserialize, PartialEq, Eq, Debug, Clone)]
+pub struct Creator {
```

Comment:
Can't most of these structs and enums be imported from mpl?

Reply:
No because Anchor won't put them in the IDL lol